### PR TITLE
Temporarily disable building the artifacts for `trigger-simnet` job

### PR DESCRIPTION
**Pull request by rcny**
`trigger-simnet` job is disabled. Disable building the relevant prerequisite artifacts too.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -254,58 +254,58 @@ generate-impl-guide:
...
+#     - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
+#     - if: $CI_COMMIT_REF_NAME == "rococo-v1"
+#       variables:
+#         <<:                        *image-variables
+#         IMAGE_NAME:                docker.io/parity/rococo
+#         DOCKER_USER:               ${Docker_Hub_User_Parity}
+#         DOCKER_PASS:               ${Docker_Hub_Pass_Parity}
+#   needs:
+#     - job:                         test-build-linux-stable
+#       artifacts:                   true
```
Comment (ordian) [inline .gitlab-ci.yml:281]: I'm not sure if this job is related to simnet, looks like we used it previously to publish rococo-v1 images among other things.

Comment (rcny) [reply .gitlab-ci.yml:281]: AFAIK @TriplEight [has introduced it](https://github.com/paritytech/polkadot/pull/2643/commits/a4420dff5f1c49aff3b29c42de051363f57592fe#diff-037ea159eb0a7cb0ac23b851e66bee30fb838ee8d0d99fa331a1ba65283d37f7R273) specifically for the simnet's needs. Anyway, I'll re-enable it as soon as I'll fix the new runner setup for such cases.

Comment (ordian) [reply .gitlab-ci.yml:281]: I'm referring to
```
#     - if: $CI_COMMIT_REF_NAME == "rococo-v1"
#       variables:
#         <<:                        *image-variables
#         IMAGE_NAME:                docker.io/parity/rococo
#         DOCKER_USER:               ${Docker_Hub_User_Parity}
#         DOCKER_PASS:               ${Docker_Hub_Pass_Parity}
#   needs:
#     - job:                         test-build-linux-stable
#       artifacts:                   true
```
part, which was introduced before that. I think we still want to publish rococo-v1 binaries as it was requested by external teams trying to test and deploy parachains.

Let's comment out only simnet part then and reeable it together with the simnet check once it's ready.

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -254,31 +254,34 @@ generate-impl-guide:
...
+  # .github/workflows/publish-docker-release.yml
+
+    # TODO: re-enable these rules when simnet is ready
+    # - if: $CI_COMMIT_REF_NAME =~ /^v[0-9]+\.[0-9]+.*$/              # i.e. v1.0, v2.1rc1
+    #   when: never
+    # - if: $CI_PIPELINE_SOURCE == "schedule"
+    # - if: $CI_COMMIT_REF_NAME == "master"
+    # - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
+
+    - if: $CI_COMMIT_REF_NAME == "rococo-v1"
```
Comment (rcny) [inline .gitlab-ci.yml:276]: @ordian Enabled `publish-polkadot-image` just for `rococo-v1`

**Review by ordian — APPROVED** [review]

**Comment by ordian** [comment]
#3021 fixes the runtime version check

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -280,32 +283,32 @@ publish-polkadot-image:
...
-  after_script:
-    - buildah logout "$IMAGE_NAME"
-    # pass artifacts to the trigger-simnet job
-    - echo "COLLATOR_IMAGE=$IMAGE_NAME" > ./artifacts/collator.env
-    - echo "COLLATOR_IMAGE_TAG=$(cat ./artifacts/EXTRATAG)" >> ./artifacts/collator.env
-  artifacts:
-    reports:
-      # this artifact is used in trigger-simnet job
-      dotenv: ./artifacts/collator.env
+# publish-adder-collator-image:
```
Comment (pepyakin) [inline .gitlab-ci.yml:286]: Nit: Perhaps we can leave a comment when we expect this to be uncommented?

**Comment by ordian** [comment]
merged as part of #3021

File: .gitlab-ci.yml (hunk 3 above)
Comment (rcny) [reply .gitlab-ci.yml:286]: `publish-adder-collator-image` is fully commented as `trigger-simnet`, implicitly implying that these should be re-enabled entirely. The TODO comment for commented part of `publish-polkadot-image` job only points to the specific rules related to the simnet stuff.

If first two commented jobs should be noted with their re-enablement condition, I can submit another PR. This one was already merged.

**Comment by rcny** [comment]
@TriplEight FYI look through this later please

**Pull request closed without merging**